db/migrate/20200404113617_add_users_to_campaigns.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToCampaigns < ActiveRecord::Migration[6.0]
+  def change
+    add_belongs_to :campaigns, :user, null: true
```

Comment:
After this is run on the server and existing campaigns are associated with a user, we can add a new migration to change this to not allow null.

Reply:
I can't see a way to invite you to the whole pipeline on heroku - do I have to create a new team for that instead?

Reply:
I'm honestly not sure. What if you add me to the app as a collaborator via the commandline? https://devcenter.heroku.com/articles/collaborating#collaborator-permissions-for-apps-in-a-personal-account

Reply:
I've created a team and invited you, the collaborators would only be per-app which wouldn't be helpful if you wanted to do anything with review apps